modules/rosa-sts-account-wide-roles-and-policies.adoc

```diff
@@ -9,6 +9,22 @@ This section provides details about the account-wide IAM roles and policies that
 
 The account-wide roles and policies are specific to an OpenShift minor release version, for example OpenShift 4.8, and are backward compatible. You can minimize the required STS resources by reusing the account-wide roles and policies for multiple clusters of the same minor version, regardless of their patch version.
 
+== Methods of account-wide role creation
```

Comment:
Should we note here that this is only relevant for using OCM to create a cluster?  Or in the contrary that this does not apply if you are only using the cli?  In reading this now, this point is not clear to me if I didnt know otherwise.